### DISCUSSION TITLE
Not able to insert images for each task.

### DISCUSSION THREAD

**Discussion by KhalidObaide** [community/community#10015] [Projects and Issues]
#### only applies for the tasks that are not converted to issues.
Users are not able to insert images into the task descriptions, unless they convert it into an issue and from that editor they can drag and drop images.

however in other project management boards like Jira, it is not only possible to insert images, but you also see the thumnail of that image on the task it's self which is really great and usefull, where just by a quick look we understand which issue I'm looking at.

**this feature is a must when doing UI.**, and some tasks cannot be converted to repositories issues.

**Comment by rmcsharry** [comment]
> #### only applies for the tasks that are not converted to issues.
> Users are not able to insert images into the task descriptions, unless they convert it into an issue and from that editor they can drag and drop images.
> 
> however in other project management boards like Jira, it is not only possible to insert images, but you also see the thumnail of that image on the task it's self which is really great and usefull, where just by a quick look we understand which issue I'm looking at.
> 
> **this feature is a must when doing UI.**, and some tasks cannot be converted to repositories issues.

![image](https://user-images.githubusercontent.com/1156100/162616291-329a6d42-4ff4-4963-8099-f69488d7ffe5.png)
